tests/storage/lvm_test.py

```diff
@@ -1694,6 +1694,32 @@ def test_vg_no_autoactivation(tmp_storage):
     assert not lvm.getLV(vg_name, lv2_name).active
 
 
+@requires_root
+@pytest.mark.root
+def test_vg_autoactivation_check_disabled(tmp_storage):
+    dev_size = 10 * GiB
+    dev = tmp_storage.create_device(dev_size)
+    vg_name = str(uuid.uuid4())
+    lv1_name = str(uuid.uuid4())
+    lv2_name = str(uuid.uuid4())
+
+    lvm.createVG(vg_name, [dev], "initial-tag", 128)
+    lvm.createLV(vg_name, lv1_name, 1024, activate=False)
+    lvm.createLV(vg_name, lv2_name, 1024, activate=False)
+
+    # Enable autoactivation for the VG.
+    commands.run(
+        ["vgchange", "--setautoactivation", "y", "--devices", dev, vg_name])
+
+    # Check should detect the VG with enabled autoactivation and disable it.
+    lvm.check_vgs_autoactivation()
+
+    # LVM command to autoactivate all LVs of the VG, but is disabled.
```

Comment:
`..., but is disabled.` I think the comment better be rephrased, or I didn't understand it, sorry then :)

Reply:
True, can be better phrased, I'll update it :)